hdfs/config.py

```diff
@@ -21,6 +20,12 @@
 import os.path as osp
 import sys
 
+try:
+  # python 3.12 and above
+  from importlib import load_source
+except ImportError:
+  # Bellow python 3.12
```

Comment:
Typo in "below" and "Python" (should be capitalized). Same in `test_examples.py`.